### PR TITLE
Fix #5973 - Disable upvoting own posts

### LINKED ISSUE #5973: Attempting to upvote own post should show error in addition to showing voting summary window
NodeBB/nodebb-theme-persona#379

### PR DESCRIPTION
**Pull request by asamolion**
Upvoting/downvoting own posts now shows a proper error message.

### CODE REVIEW

File: public/language/en-GB/error.json [hunk 1]
```diff
@@ -144,6 +144,7 @@
 	"not-enough-reputation-to-downvote": "You do not have enough reputation to downvote this post",
 	"not-enough-reputation-to-flag": "You do not have enough reputation to flag this post",
 	"already-flagged": "You have already flagged this post",
+	"self-vote": "You cannot upvote/downvote your own post",
```
Comment (pitaj) [inline public/language/en-GB/error.json]: "You cannot vote on your own post" would be better

**Comment by julianlam** [comment]
Merged w/ error copy changes.

**Comment by asamolion** [comment]
Thanks. Sorry, I didn't have time to make the final edit in the error message.

**Comment by aStonedPenguin** [comment]
This should have a config too imo

**Comment by asamolion** [comment]
@aStonedPenguin Can you please elaborate on what type of config?

**Comment by aStonedPenguin** [comment]
To allow users to upvote their own posts

**Comment by julianlam** [comment]
That is a separate issue and not likely one we'll change

**Comment by psychobunny** [comment]
Facebook allows it, and our "like" plugin that switches out upvotes for that ends up with this restriction.

I'll have to respectfully disagree on this one here. Separate issue though.

**Pull request merged**